src/emitter.js

```diff
@@ -1,22 +1,23 @@
 /**
+ * @flow
  * @fileoverview Emitter is an abstract class used by a few other
  * classes to communicate via events
  */
 
 import { isFunction } from './utils'
 
+type EventName = string
+type EventPayload = Array<any>
+type EventCallback = (...payload: EventPayload) => {}
```

Comment:
Wondering if we shouldn't pull all these type definitions into a `flow` folder or a `flow-typed` folder. Like [this](https://flow.org/en/docs/libdefs/creation/) suggests (I think)  or the way [vue.js does it](https://github.com/vuejs/vue/tree/dev/flow).